lib/gli/commands/scaffold.rb

```diff
@@ -31,6 +31,7 @@ def self.create_scaffold(root_dir,
           end
           puts "Created #{rvmrc}"
         end
+        Dir.mkdir(File.join(root_dir, project_name, ".git"))
```

Comment:
I'm not sure what will happen if you run `git init` after creating the directory.  Should we maybe do a `git init` instead?

Reply:
You're absolutely right. For some reason I was thinking that `git init` just creates the empty dir, but there's a lot more. I've changed it. I run the command in `quiet` mode because I thought it wasn't really important to make it explicit in the output.